### ISSUE TITLE
Doesn't work with Ruby 2.3?

### ISSUE THREAD

**Issue by oniofchaos** [brigade/haml-lint#122]
When using Ruby 2.3, I get the following error:

```ruby
Dillons-MacBook-Air:web dillonwelch$ haml-lint
/Users/dillonwelch/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/specification.rb:2158:in `method_missing': undefined method `this' for #<Gem::Specification:0x3fc1b286f3a4 haml_lint-0.16.1> (NoMethodError)
	from /Users/dillonwelch/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/specification.rb:1057:in `find_active_stub_by_path'
	from /Users/dillonwelch/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:64:in `require'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.0/gems/haml_lint-0.16.1/bin/haml-lint:3:in `<top (required)>'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.0/bin/haml-lint:23:in `load'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.0/bin/haml-lint:23:in `<main>'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `eval'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `<main>'
```

When I switch back to Ruby 2.2.3, it works fine.

Am I doing something silly? Maybe a dependency I didn't install for Ruby 2.3? :cry:

**Comment by sds** [comment]
Hey @oniofchaos, thanks for the report.

This is not is an issue with HAML-Lint since it's failing on the line `require 'haml_lint'`, which means there's an internal Ruby issue here.

It looks like https://github.com/rubygems/rubygems/issues/1420 is relevant to your particular issue. Perhaps you should try downgrading Rubygems as suggested in https://github.com/rubygems/rubygems/issues/1420#issuecomment-167125657. 

Otherwise, I don't think there's anything HAML-Lint can do here to help. I (only half-jokingly) suggest you switch to `rbenv` instead of `rvm`, as I see these kinds of issues only with the latter.

**Comment by oniofchaos** [comment]
Great, thanks for the heads up re: RubyGems. :cake:
